File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum,int pageSize);

    public void save(TbBrand brand);

    public TbBrand findById(Long id);

    public void update(TbBrand brand);

    public void delete(Long[] ids);

    public PageResult findPage(TbBrand brand,int pageNum,int pageSize);

    List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findPage = function (page, rows) {
        return $http.get("../brand/findPage.do?page=" + page + "&rows=" + rows);
    };

    this.findAll=function () {
        return $http.get("../brand/findAll.do");
    };
    
    this.add=function (entity) {
        return $http.post("../brand/add.do",entity);
    };
    this.update = function (entity) {
        return $http.post("../brand/update.do",entity)
    };

    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id=" + id);
    };

    this.dele = function (Ids) {
       return $http.get("../brand/delete.do?ids=" +Ids);
    };

    this.search = function (page, rows,searchEntity) {
        return $http.post("../brand/search.do?page=" + page + "&rows="+ rows,searchEntity);
    }

    this.selectOptionList = function () {
        return $http.get("../brand/selectOptionList.do");
    }
});

